ftplugin/org.vim

```diff
@@ -104,17 +120,24 @@ function! OrgExportToHTML()
     let l:out = system(join(l:cmd))
 
     if v:shell_error != 0
-        echoerr l:out
+        OrgEchoError(l:out)
     endif
 endfunction
 
 function! OrgMakeProgn(prog)
-    if type(a:prog) == v:t_list
-        let l:prog = join(a:prog, ' ')
-    else
-        let l:prog = a:prog
+    " Support the different kinds of way to define a program (string, list,
```

Comment:
It is not clear to me how this is used. Would you provide some context on what 'program' you're talking about?

Reply:
Is this the org program that updates tables and stuff?

Reply:
This is going to be refactored in #31, then I should document it better. In any case, I've opened #32 too.